lib/blocks/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@storybook/blocks",
+  "version": "7.0.0-alpha.6",
+  "description": "Storybook Doc Blocks",
+  "keywords": [
+    "storybook"
+  ],
+  "homepage": "https://github.com/storybookjs/storybook/tree/main/lib/blocks",
+  "bugs": {
+    "url": "https://github.com/storybookjs/storybook/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybookjs/storybook.git",
+    "directory": "lib/blocks"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/storybook"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "prepare": "node ../../scripts/prepare.js"
+  },
+  "dependencies": {
+    "@mdx-js/react": "^1.6.22",
+    "@storybook/addons": "7.0.0-alpha.6",
+    "@storybook/api": "7.0.0-alpha.6",
+    "@storybook/client-logger": "7.0.0-alpha.6",
+    "@storybook/components": "7.0.0-alpha.6",
+    "@storybook/core-events": "7.0.0-alpha.6",
+    "@storybook/csf": "0.0.2--canary.7c6c115.0",
+    "@storybook/docs-tools": "7.0.0-alpha.6",
+    "@storybook/preview-web": "7.0.0-alpha.6",
+    "@storybook/store": "7.0.0-alpha.6",
+    "@storybook/theming": "7.0.0-alpha.6",
```

Comment:
How are we thinking about the dependencies of this package? We might need to discuss @shilman. In particular I am interested in the thinking behind pulling some but not all of the dependencies from `@storybook/components` into this package.

Reply:
Ideally this doesn't pull anything form `@storybook/components`, but I think we'll need to tackle that later on.